model/Major.js

```diff
@@ -25,19 +25,19 @@ const majorSchema = new Schema({
       name: { type: String, required: true },
```

Comment:
line 18: degree_name should be the id of the major
remove line 18
add `_id: {type: String, required: true}`